.changeset/crazy-doors-buy.md

```diff
@@ -0,0 +1,26 @@
+---
+'astro': minor
+---
+
+Adds the experimental support of Content Security Protection (CSP) in Astro. 
+
+CSP is an important feature for websites that require fine-grained control over the resources that can or cannot be downloaded and executed.
```

Comment:
```suggestion
CSP is an important feature to provide fine-grained control over resources that can or cannot be downloaded and executed by a document. In particular, it can help protect against [cross-site scripting (XSS)](https://developer.mozilla.org/en-US/docs/Glossary/Cross-site_scripting) attacks.
```